ngraph_creator/operations/src/ResizeBilinear.cpp

```diff
@@ -0,0 +1,136 @@
+#include <ResizeBilinear.hpp>
+#define LOG_TAG "ResizeBilinear"
+
+namespace android {
+namespace hardware {
+namespace neuralnetworks {
+namespace nnhal {
+
+ResizeBilinear::ResizeBilinear(int operationIndex) : OperationsBase(operationIndex) {
+    mDefaultOutputIndex = sModelInfo->getOperationOutput(mNnapiOperationIndex, 0);
+}
+
+bool ResizeBilinear::validate() {
+    // TODO Add FLOAT16 check when VPUX plugin is supported
+    if (!checkOutputOperandType(0, (int32_t)OperandType::TENSOR_FLOAT32) &&
+        !checkOutputOperandType(0, (int32_t)OperandType::TENSOR_QUANT8_ASYMM)) {
+        ALOGE("%s check for output types failed", __func__);
+        return false;
+    }
+
+    if (!checkInputOperandType(0, (int32_t)OperandType::TENSOR_FLOAT32) &&
+        !checkInputOperandType(0, (int32_t)OperandType::TENSOR_QUANT8_ASYMM)) {
+        return false;
+    }
+
+    const auto& inputDimensionsSize = getInputOperandDimensions(0).size();
+    if (inputDimensionsSize != 4) {
+        ALOGE("%s Invalid dimensions size for input(%d)", __func__, inputDimensionsSize);
+        return false;
+    }
+    ALOGV("%s PASSED", __func__);
+    return true;
+}
+
+std::shared_ptr<ngraph::Node> ResizeBilinear::createNode() {
+    const auto& inputsSize = sModelInfo->getOperationInputsSize(mNnapiOperationIndex);
+
+    std::shared_ptr<ngraph::Node> outputNode;
+    int32_t input_width, input_height;
+    bool useNchw = false;
+    int32_t layout = 0;
+    float width_scale, height_scale;
+    bool align_corners = false, half_pixel = false;
+    const auto& inputDimensions = getInputOperandDimensions(0);
+    int32_t out_width, out_height;
+
+    std::shared_ptr<ngraph::Node> inputNode;
+    struct ngraph::op::v4::Interpolate::InterpolateAttrs attrs;
+
+    inputNode = getInputNode(0);
+    switch (inputsSize) {
+        case 6:
+            half_pixel = sModelInfo->ParseOperationInput<uint8_t>(mNnapiOperationIndex, 5);
+        case 5:
+            align_corners = sModelInfo->ParseOperationInput<uint8_t>(mNnapiOperationIndex, 4);
+        case 4:
+            layout = sModelInfo->ParseOperationInput<uint8_t>(mNnapiOperationIndex, 3);
+        default:
+            break;
+    }
```

Comment:
This results in compiler warnings. Can u look into adding appropriate flags to remove the warnings.

Reply:
Updated code to remove warnings.

Reply:
Updated code to remove warnings